<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="CHNMCT369326L" failures="0" tests="3" name="com.testCases.TestFlow" time="7.794" errors="1" timestamp="07 Aug 2020 04:09:38 GMT" skipped="2">
  <testcase classname="com.testCases.TestFlow" name="TC_001_AppleLaptopFunctionality" time="0.000">
    <skipped/>
  </testcase> <!-- TC_001_AppleLaptopFunctionality -->
  <testcase classname="com.testCases.TestFlow" name="TC_002_AppleLaptopFunctionality" time="0.000">
    <skipped/>
  </testcase> <!-- TC_002_AppleLaptopFunctionality -->
  <testcase classname="com.testCases.TestFlow" name="beforeMethod" time="7.794">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at utils.Reporter.reportStep(Reporter.java:59)
at utils.Reporter.reportStep(Reporter.java:64)
at com.amazon.selenium.base.SeleniumBase.startApp(SeleniumBase.java:278)
at com.qa.ApplicationSpecifics.ProjectSpecificMethods.beforeMethod(ProjectSpecificMethods.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:523)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- beforeMethod -->
</testsuite> <!-- com.testCases.TestFlow -->
